docker/base.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+DOCKER_COMPOSE_BIN="docker-compose"
+DOCKER_COMPOSE_FROM_DOCKER="docker compose"
+DOCKER_COMPOSE=""
```

Comment:
instead of storing a variable, what you think about echoing the value?

Reply:
First, to be honest, not sure if is really needed this line 5, I could remove that. 

----

Second, I can do this here `base.sh`

```bash
if [ -x "$(command -v $DOCKER_COMPOSE_BIN)" ]; then
  echo $DOCKER_COMPOSE_BIN
else
  echo $DOCKER_COMPOSE_FROM_DOCKER
fi
```

And on `down.sh and run.sh`

```bash
DOCKER_COMPOSE=$("$ROOT/docker/base.sh")
```
Works!

----
My concern with this change is that the file's name is `base`, and the idea is to add/move more things into it, for example, `$ROOT/docker/dockerfile` that has been repeated many times, but the value is the same for all usages.

So, to return more than one value from `base` would be difficult or I just don't know how to do it.

So, let me know what is your idea and let's make that happen :)

Reply:
Should I echo the value instead assigned into a variable, then?

Reply:
Still did not have time to review this one yet :grimacing: 

Once I review I will answer you!